howto/stream/oob-data.md

```diff
@@ -103,4 +123,97 @@ of the `AnboxStream` object:
         }
       }
     });
-```
\ No newline at end of file
+```
+
+<a name="oob-v2"></a>
+## Version 2
+
+The following instructions will walk you through how to set up data channels and perform data transmission in both directions between an Android application and a WebRTC platform.
+
+### Web application
+
+In your web-based client application, import the [Anbox Streaming SDK](https://discourse.ubuntu.com/t/anbox-cloud-sdks/17844#streaming-sdk).
+
+Create a data channel (named `foo` in the following example) under the `dataChannels` property of an `AnboxStream` object and register event handlers that respond to the events sent from an Android application:
+
+```
+    let stream = new AnboxStream({
+      ...
+      ...
+      dataChannels: {
+        "foo": {
+          callbacks: {
+            close: () => {
+              console.log('data channel is closed')
+            },
+            open: () => {
+              console.log('data channel is open')
+            },
+            error: (err) => {
+              console.log(`error: ${err}`)
+            },
+            message: (data) => {
+              console.log(`data received: ${data}`)
+            }
+          }
+        }
+      }
+    });
+```
+
+[note type="information" status="Note"]
+An `AnboxStream` object can create a maximum of five data channels. If the number of data channels exceeds the allowed maximum, an exception is thrown when instantiating the `AnboxStream` object.
+[/note]
+
+To launch a new WebRTC session, the client must call `stream.connect()`. The `AnboxStream` object then builds up a WebRTC native data channel internally, based on the name declared under its `dataChannels` property for peer-to-peer communication (`foo` in the example).
+
+To send data to an Android application through the channel, the client must use the member function `sendData` of the `AnboxStream` class:
+
+```
+        stream.sendData('foo', 'hello world');
+```
+
+
+### Anbox WebRTC platform
+
+When establishing a peer connection, a number of data channels are set up in the Anbox WebRTC platform on the server side, upon request by the client.
+
+At the same time, a number of Unix domain sockets are created under the `/run/user/1000/anbox/sockets` folder. They use the format `webrtc_data_<channel_name>` and represent the established communication bridge between a WebRTC client and the Anbox WebRTC platform. Those Unix domain sockets can be used by a service or daemon to:
+
+- Receive the data sent from a WebRTC client over the data channel and forward it to an Android application.
+- Receive data sent from an Android application and forward it to a WebRTC client over the data channel.
```

Comment:
I missed a `the` here.
`Receive the data ....`

Reply:
It actually sounds a bit better without the "the" (since we're talking about any data), so removing it for both. :)